Geometry/HGCalCommonData/plugins/DDHGCalHEAlgo.cc

```diff
@@ -336,15 +356,14 @@ void DDHGCalHEAlgo::positionMix(const DDLogicalPart& glog,
   // Make the top part first
   std::string name = nameM+"Top";
   DDSolid solid = DDSolidFactory::tubs(DDName(name, nameSpace_), 
-				       hthick, rmid, rout, 0.0,
-				       CLHEP::twopi);
+				       hthick, rmid, rout, 0.0, 2._pi);
   glog1 = DDLogicalPart(solid.ddname(), matter, solid);
 #ifdef EDM_ML_DEBUG
   edm::LogVerbatim("HGCalGeom") << "DDHGCalHEAlgo: " << solid.name() 
 				<< " Tubs made of " << matter.name() 
 				<< " of dimensions " << rmid << ", " << rout 
 				<< ", " << hthick << ", 0.0, " 
-				<< CLHEP::twopi/CLHEP::deg;
+				<< convertRadToDeg(2._pi);
```

Comment:
Same comment as before. Is `2._pi` acting as a magic number here?

Reply:
Done. But 2._piis indeed a magic number - the total angular coverage is 2._pi